pyke/kepclean.py

```diff
@@ -88,7 +88,7 @@ def kepclean(infile, outfile=None, zero=True, overwrite=False, verbose=False,
                 except:
                     continue
         flux_keys = np.asarray(flux_keys,dtype=bool)
-        if np.any([(len(flux_keys) == 0),(len(np.where(flux_keys==True)[0]) == 0)]) :
+        if np.any([(len(flux_keys) == 0),(len(np.where(flux_keys == True)[0]) == 0)]) :
```

Comment:
Add a blank space after the comma here. No need to a add a blank space before ``:``.
Also, change ``np.where(flux_keys == True)`` to ``np.where(flux_keys)``.